health_ping.py

```diff
@@ -22,6 +22,19 @@
 refresh_interval = int(os.getenv("REFRESH_INTERVAL","60"))
 log_level = os.environ.get('LOG_LEVEL', 'INFO').upper()
 
+# limit results for testing/dev
+# See strapi filter syntax https://docs.strapi.io/dev-docs/api/rest/filters-locale-publication
+# Example filter string = '&filters[name][$contains]=example'
+# sc_filter = '&filters[name][$contains]=book-a-prison-visit-staff-ui'
+sc_filter = os.getenv("sc_filter")
+sc_page_size = 10
+sc_pagination_page_size = f"&pagination[pageSize]={sc_page_size}"
```

Comment:
the pagination variables will not work with out other code to iterate over the pages. Suggest you remove this block that has been copy/pasted.